assets/js/components/notifications/BannerNotification/index.js

```diff
@@ -525,12 +346,10 @@ BannerNotification.propTypes = {
 	className: PropTypes.string,
 	title: PropTypes.string.isRequired,
 	description: PropTypes.node,
-	descriptionIcon: PropTypes.node,
 	learnMoreURL: PropTypes.string,
 	learnMoreDescription: PropTypes.string,
 	learnMoreLabel: PropTypes.string,
 	learnMoreTarget: PropTypes.oneOf( Object.values( LEARN_MORE_TARGET ) ),
-	blockData: PropTypes.array,
```

Comment:
It's great to remove this to reduce complexity. Can I just confirm we have no more references to `blockData` on the backend?

I can't see any real danger of it being relied on in the client-side anymore anyway... The only place we're spreading props into `BannerNotification` is in `InternalServerError` and I can't see an error object containing a `blockData` property!